repos/fdbt-admin/src/App.tsx

```diff
@@ -11,29 +10,15 @@ import Reporting from './pages/Reporting';
 import ListUsers from './pages/ListUsers';
 import ResendInvite from './pages/ResendInvite';
 
-interface AuthDataAttributes {
-    'custom:fullAdmin': number;
-}
-
-interface AuthData {
-    attributes: AuthDataAttributes;
-}
-
-const App = (): ReactElement => {
```

Comment:
reporting tab is giving Access to XMLHttpRequest at 'https://fdbt-products-data-test.s3.eu-west-2.amazonaws.com/?list-type=2' from origin 'http://localhost:5173' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.